<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5FCAB7O" failures="2" tests="7" name="BankingApp Test" time="76.596" errors="0" timestamp="03 Sep 2019 02:29:53 GMT">
  <testcase classname="com.inetbanking.testCases.TC_LoginTest_001" name="loginTest" time="2.237"/>
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="9.056"/>
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="4.238">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:32)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="8.755"/>
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="4.547">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:32)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="8.603"/>
  <testcase classname="com.inetbanking.testCases.TC_AddCustomerTest_003" name="addNewCustomer" time="16.499"/>
</testsuite> <!-- BankingApp Test -->
